labelbox/schema/model_run.py

```diff
@@ -175,6 +181,60 @@ def delete_model_run_data_rows(self, data_row_ids):
             data_row_ids_param: data_row_ids
         })
 
+    @experimental
+    def export_annotations(self, download: bool = False) -> LabelGenerator:
+        """
+        Experimental. To use, make sure client has enable_experimental=True.
+
+        Fetches Labels from the ModelRun
+
+        Args:
+            download (bool): Returns the url if False
+        Returns:
+            URL of the data file that is generated from this ModelRun. 
+            If download=True, this instead returns the contents as NDJSON format.
+        """
+
+        query_str = """
+            mutation exportModelRunAnnotationsPyApi($modelRunId: ID!) {
+                exportModelRunAnnotations(data: {modelRunId: $modelRunId}) {
+                    downloadUrl createdAt status
+                }
+            }
+            """
+        url = self.client.execute(
+            query_str, {'modelRunId': self.model_id},
+            experimental=True)['exportModelRunAnnotations']['downloadUrl']
+
+        counter = 1
+        while url is None:
+            logger.info(f"Fetching model run annotations...attempt: {counter}")
+            counter += 1
+            if counter == 10:
+                raise Exception(
+                    f"Unsuccessfully got downloadUrl after {counter} attempts.")
+            time.sleep(10)
+            url = self.client.execute(
+                query_str, {'modelRunId': self.model_id},
+                experimental=True)['exportModelRunAnnotations']['downloadUrl']
+
+        if not download:
+            return url
+        else:
+            response = requests.get(url)
+            response.raise_for_status()
+            contents = ndjson.loads(response.content)
+            return contents
```

Comment:
I would just copy this as closely as possible:
https://github.com/Labelbox/labelbox-python/blob/ac1a03739ddcfe480416bf0ef169c1addfcf79a4/labelbox/schema/project.py#L342-L364